sys/mips/beri/files.beri

```diff
@@ -26,6 +26,7 @@ dev/terasic/pixelstream/pixelstream.c		optional pixelstream
 dev/terasic/pixelstream/pixelstream_fdt.c	optional pixelstream
 dev/terasic/pixelstream/pixelstream_reg.c	optional pixelstream
 dev/terasic/pixelstream/pixelstream_vt.c	optional pixelstream vt
+mips/beri/asm_beri.S			standard
```

Comment:
```suggestion
mips/beri/asm_beri.S			optional smp
```

Reply:
Yes, and I renamed the file to `asm_beri_mp.S` to reflect the intent.